content/buildah-podman-skopeo.howto.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Building Containers safely without Docker (Look Ma! No Docker!)"
+date: 2020-09-29T13:42:36-07:00 
+draft: true
+---
+
+If you want to create a container, the traditional way to go about it is to use just `docker build .` and let docker take care of it for you.
+
+One problem with this is that it requires access to the Docker daemon which then runs as root. It also tends to lead to more bloated images as the typical Dockerfile starts with pulling in someone elses' base container (not always bad, just be aware of where your base containers come from).
```

Comment:
There's a million reasons why one would want something else than docker. The daemon is certainly one. How about a slightly broader approach like: "But docker is for sure not the only way to build containers and, although the alternatives are much less broadly covered, they are actually no less convenient."

Reply:
I'd rather stick to a couple tangible reasons